### DISCUSSION TITLE
EasyEffects Preset Location

### DISCUSSION THREAD

**Discussion by robotard** [wwmm/easyeffects#1258] [General]
Hey guys, where are you hiding the EasyEffects presets folder now?

They used to be in ~/.config/PulseEffects/output.

Also the old ones from PulseEffects, upon importing seem to have weird/different results in Easy with PipeWire-Pulse...

**Comment by wwmm** [comment]
Presets and impulse response files are under `~/.config/easyeffects/`. But have in mind that some changes had to be made to the presets structure when writing EasyEffects. There is a site that can convert them. Look at the last section of https://github.com/wwmm/easyeffects/wiki/Community-Presets.

**Reply by robotard** [reply]
Strangely I don't appear to have an EasyEffects folder, but also need to bounce my box for flatpak apparently... Have been launching directly from flatpak run whilst testing it all... 

That link looks useful though... Was manually opening the json files for a couple purely for a little test, but indeed the PulseEffects ones do break EasyEffects lol. Improving things sometimes simply has to come with consequences...

**Comment by wwmm** [comment]
>need to bounce my box for flatpak apparently

The flatpak package saves in a different folder. I do not remember now which one.

**Reply by robotard** [reply]
Yes, I couldn't see an apt installer... I don't really use flatpak usually... 

Is there an option for that or is it the case of dpkg?

**Reply by vchernin** [reply]
`~/.var/app/com.github.wwmm.easyeffects/config/easyeffects` should be equivalent to `~/.config/easyeffects/`

#1000 tracks building with apt/dpkg.

**Comment by robotard** [comment]
Ah, those long (flat)package names though... Are there any thoughts to offer apt too?

Was kinda hoping all that namespace stuff had been sorted ages back.... 

Super impressed, though, that you guys pulled a piperwire outta the bag to align with 21.10/Wayland/Gnoe 4 etc....

**Reply by wwmm** [reply]
>Super impressed, though, that you guys pulled a piperwire outta the bag to align with 21.10/Wayland/Gnoe 4 etc....

Thanks! Although the only thing I really had in my mind was the gtk4 release. PipeWire support was something people asked some years ago.

**Comment by wwmm** [comment]
>Are there any thoughts to offer apt too?

There is an issue about Debian packages at #1000. Long story short as I do not use the system someone that does has to help. And probably create a PPA repository with the dependencies Debian systems do not have now. This task should get easier once I finish the work I am doing right now on #1243.

**Comment by robotard** [comment]
> There is a site that can convert them
Awesome work by the community here, as always :)

The site worked great, however the batch script gave me 0 byte files... I've only got 7 of them, so all in all, not a big deal... Would be cool to have this a post installation option moving the old ones to /pe-bak or the likes, though given that I only ever use the /output folder, I can imagine there's likely other work here needed to be done...
